src/js/button.js

```diff
@@ -1,7 +1,9 @@
 import Component from './component';
-import * as Lib from './lib';
-import * as Events from './events';
+import * as Dom from './utils/dom.js';
+import * as Events from './utils/events.js';
+import * as Fn from './utils/fn.js';
 import document from 'global/document';
+import assign from 'object.assign';
```

Comment:
we probably should change this assign to be something like
```js
import objectAssign from 'object.assign';
let assign = Object.assign || objectAssign;
```
We could even have a small internal module that does this wrapping. This is because the npm module doesn't automatically change over to the built-in once available.